cryptofeed/exchanges/binance.py

```diff
@@ -470,6 +471,57 @@ async def _account_update(self, msg: dict, timestamp: float):
                                 receipt_timestamp=timestamp,
                                 wallet_balance=Decimal(balance['f']))
 
+    async def _order_update(self, msg: dict, timestamp: float):
+        """
+        {
+            "e": "executionReport",        // Event type
+            "E": 1499405658658,            // Event time
+            "s": "ETHBTC",                 // Symbol
+            "c": "mUvoqJxFIILMdfAW5iGSOW", // Client order ID
+            "S": "BUY",                    // Side
+            "o": "LIMIT",                  // Order type
+            "f": "GTC",                    // Time in force
+            "q": "1.00000000",             // Order quantity
+            "p": "0.10264410",             // Order price
+            "P": "0.00000000",             // Stop price
+            "F": "0.00000000",             // Iceberg quantity
+            "g": -1,                       // OrderListId
+            "C": "",                       // Original client order ID; This is the ID of the order being canceled
+            "x": "NEW",                    // Current execution type
+            "X": "NEW",                    // Current order status
+            "r": "NONE",                   // Order reject reason; will be an error code.
+            "i": 4293153,                  // Order ID
+            "l": "0.00000000",             // Last executed quantity
+            "z": "0.00000000",             // Cumulative filled quantity
+            "L": "0.00000000",             // Last executed price
+            "n": "0",                      // Commission amount
+            "N": null,                     // Commission asset
+            "T": 1499405658657,            // Transaction time
+            "t": -1,                       // Trade ID
+            "I": 8641984,                  // Ignore
+            "w": true,                     // Is the order on the book?
+            "m": false,                    // Is this trade the maker side?
+            "M": false,                    // Ignore
+            "O": 1499405658657,            // Order creation time
+            "Z": "0.00000000",             // Cumulative quote asset transacted quantity
+            "Y": "0.00000000",             // Last quote asset transacted quantity (i.e. lastPrice * lastQty)
+            "Q": "0.00000000"              // Quote Order Qty
+        }
+        """
+        oi = OrderInfo(
+            self.id,
+            self.exchange_symbol_to_std_symbol(msg['s']),
+            msg['i'],
+            BUY if msg['S'].lower() == 'buy' else SELL,
+            msg['x'],
+            LIMIT if msg['o'].lower() == 'limit' else MARKET if msg['o'].lower() == 'market' else None,
+            Decimal(msg['Z'] / Decimal(msg['z'])) if not Decimal.is_zero(Decimal(msg['z'])) else None,
+            Decimal(msg['q']),
+            Decimal(msg['q']) - Decimal(msg['z']),
+            self.timestamp_normalize(msg['E']),
+        )
```

Comment:
only thing missing would be to add the raw kwarg and store the raw message, but I can add this

Reply:
Thanks Bryant!